### ISSUE TITLE
Idea: @AndroidInstance field annotation to persist via onSaveRestoreInstanceState

### ISSUE THREAD

**Issue by Buggaboo** [tobykurien/Xtendroid#132]

**Comment by tobykurien** [comment]
Nice idea! I generally work around the need to save/restore state by using bundle arguments to fragments and letting the framework do the rest.

**Comment by Buggaboo** [comment]
Yeah, me too :) But since we can't change the state of bundle after the first lifecycle methods (that checks for changes and throws `IllegalStateException`, we're still relegated to the standard way of persisting objects through `on*InstanceState`. Then again, I think it's better to let the Activity or the P in (MVP) do the networking and commit fragment tx's.
